<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigEditor</class>
 <widget class="QWidget" name="ConfigEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="labTitle">
       <property name="text">
        <string>标题        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineTitle"/>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>主界面背景  </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="lineBack"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBackSp">
         <property name="text">
          <string>特殊</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnBack">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>图标        </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineIcon"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnIcon">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>主界面音乐  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineBgm"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnBgm">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="labSize">
       <property name="text">
        <string>存档界面背景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSave"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnSave">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="text">
        <string>全局配置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="win_save">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>读档界面背景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineLoad"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnLoad">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>画廊界面背景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineGal"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnGal">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>分辨率      </string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QSpinBox" name="spinSizeX">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>1999</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinSizeY">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>1999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="10" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>设置界面背景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineCon"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnCon">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>鼠标样式    </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineMouse"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnMouse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
